src/Tests/Microsoft.Diagnostics.Monitoring.UnitTests/AuthenticationTests.cs

```diff
@@ -185,9 +187,11 @@ public async Task ApiKeyAuthenticationSchemeTest()
         /// <summary>
         /// Tests that Negotiate authentication can be used for authentication.
         /// </summary>
-        [Fact]
+        [SkippableFact]
```

Comment:
The other option you can have here is to reference `Xunit.SkippableFact@1.4.13` if you don't want to maintain the fork of the code. There's also a way to get the diagnostics version of this code, but I don't know if it adds more value than the other package.